tests/L0/run_amp/test_basic_casts.py

```diff
@@ -73,10 +73,12 @@ def setUp(self):
 
     def tearDown(self):
         self.handle._deactivate()
-
+    
+    @unittest.skip("The failing unit test is introduced by new PyTorch commit as of 12/1/2021. Same error is also observed on CUDA.")
```

Comment:
@hubertlu-tw I filed two Apex issues: #62 and #63 to track these failures. Can you please mention these issues in the skip reason instead of the current skip reason?